src/java/io/bazel/rulesscala/test_discovery/FilteredRunnerBuilder.scala

```diff
@@ -11,17 +11,14 @@ import scala.collection.JavaConverters._
 
 class FilteredRunnerBuilder(builder: RunnerBuilder, filteringRunnerBuilder: FilteringRunnerBuilder) extends RunnerBuilder {
   // Defined by --test_filter bazel flag.
-  private val maybePattern = sys.env.get("TESTBRIDGE_TEST_ONLY").map(Pattern.compile)
-
-  def runnerFilter(runner: Runner, testClass: Class[_], pattern: Pattern): Runner = runner
```

Comment:
I think I’m missing something.
Why was this removed?
Also, why did you add type annotation to the below line?

Reply:
This function wasn't used anywhere, I don't even know why it was added in the first place.
And the type annotation is a leftover, I can remove it.